CHANGELOG.md

```diff
@@ -1,5 +1,716 @@
+  - fix go vet and staticcheck errors ([multiformats/go-multistream#60](https://github.com/multiformats/go-multistream/pull/60))
+  - Implement simultaneous open extension ([multiformats/go-multistream#42](https://github.com/multiformats/go-multistream/pull/42))
+  - reduce the number of streams in the stress tests, fix error handling ([multiformats/go-multistream#54](https://github.com/multiformats/go-multistream/pull/54))
+- github.com/whyrusleeping/cbor-gen (v0.0.0-20200710004633-5379fc63235d -> v0.0.0-20210219115102-f37d292932f2):
+  - feat: allow unmarshaling of struct with more fields than marshaled struct ([whyrusleeping/cbor-gen#50](https://github.com/whyrusleeping/cbor-gen/pull/50))
+  - chore: add a license file ([whyrusleeping/cbor-gen#49](https://github.com/whyrusleeping/cbor-gen/pull/49))
+  - fix: enforce maxlen in ReadByteArray() ([whyrusleeping/cbor-gen#43](https://github.com/whyrusleeping/cbor-gen/pull/43))
+  - use unix nanoseconds for encoding Cbortime ([whyrusleeping/cbor-gen#41](https://github.com/whyrusleeping/cbor-gen/pull/41))
+  - add json marshalers to CborTime
+  - add a helper for roundtripping time.time objects ([whyrusleeping/cbor-gen#40](https://github.com/whyrusleeping/cbor-gen/pull/40))
+  - Add a validate function. ([whyrusleeping/cbor-gen#39](https://github.com/whyrusleeping/cbor-gen/pull/39))
+  - Fix import handling ([whyrusleeping/cbor-gen#38](https://github.com/whyrusleeping/cbor-gen/pull/38))
+  - Optimize discarding in ScanForLinks ([whyrusleeping/cbor-gen#36](https://github.com/whyrusleeping/cbor-gen/pull/36))
+  - Always allocate scratch space when marshalling into a map. ([whyrusleeping/cbor-gen#37](https://github.com/whyrusleeping/cbor-gen/pull/37))
+  - optimize byte reading ([whyrusleeping/cbor-gen#35](https://github.com/whyrusleeping/cbor-gen/pull/35))
+  - Optimize decoding ([whyrusleeping/cbor-gen#34](https://github.com/whyrusleeping/cbor-gen/pull/34))
+  - Fix named string issue ([whyrusleeping/cbor-gen#30](https://github.com/whyrusleeping/cbor-gen/pull/30))
+  - Fix encoding/decoding fixed byte arrays ([whyrusleeping/cbor-gen#29](https://github.com/whyrusleeping/cbor-gen/pull/29))
+  - fix overread on scanforlinks ([whyrusleeping/cbor-gen#28](https://github.com/whyrusleeping/cbor-gen/pull/28))
+
+### ❤️ Contributors
+
+| Contributor | Commits | Lines ± | Files Changed |
+|-------------|---------|---------|---------------|
+| Marten Seemann | 358 | +17444/-12000 | 1268 |
+| Eric Myhre | 82 | +9672/-2459 | 328 |
+| Ian Davis | 7 | +8421/-737 | 116 |
+| Daniel Martí | 18 | +2733/-4377 | 313 |
+| Adin Schmahmann | 46 | +5387/-1289 | 125 |
+| Steven Allen | 95 | +3278/-1861 | 200 |
+| hannahhoward | 14 | +1380/-3667 | 84 |
+| gammazero | 29 | +2520/-1161 | 88 |
+| Hector Sanjuan | 12 | +511/-3129 | 52 |
+| vyzo | 77 | +2198/-940 | 117 |
+| web3-bot | 52 | +1161/-359 | 65 |
```

Comment:
Not sure how we normally deal with bots in the contributor list, but we might want to remove this entry.

Reply:
Ya, makes sense I'll just manually drop it for now

Reply:
FWIW it's not like this list is totally accurate anyway (includes commits moved between repos, misses new repos, does ok but not great accounting for codegened files, etc.) so not fixing this isn't a big deal